test/core/domainerTests.ts

```diff
@@ -11,7 +11,9 @@ describe("Domainer", () => {
   });
 
   it("pad() works in general case", () => {
-    scale._updateExtent("1", "x", [100, 200]);
+    var provider: Plottable.Scales.ExtentProvider<number> = (scale: Plottable.Scale<number, number>) => [[100, 200]];
```

Comment:
I think the compiler was annoyed at me for some reason, but at any rate it won't hurt.